apps/signer/main.c

```diff
@@ -102,14 +102,14 @@ main(gint argc, gchar *argv[])
       "Usage:\n%s [-h] [-c codec] filename\n\n"
       "Optional\n"
       "  -c codec  : 'h264' (default) or 'h265'\n"
-      "  -r recurrence  : '1' (default) or (1-2147483647)\n"
+      "  -r recurrence  : '0' (default) or (1-2147483647)\n"
```

Comment:
This is actually not correct. There is no default value. We rely on the default value in signed-video-framework, since if the user do not set the flag we do not call the function. Please remove the default stuff.